files/zh-tw/web/http/index.md

```diff
@@ -33,24 +34,24 @@ slug: Web/HTTP
 
 詳細的 HTTP 參考文件。
 
-- [HTTP Headers](/zh-TW/docs/Web/HTTP/Headers)
-  - : HTTP message headers are used to describe a resource, or the behavior of the server or the client. Custom proprietary headers can be added using the `X-` prefix; others in an [IANA registry](http://www.iana.org/assignments/message-headers/perm-headers.html), whose original content was defined in [RFC 4229](http://tools.ietf.org/html/rfc4229). IANA also maintains a [registry of proposed new HTTP message headers](http://www.iana.org/assignments/message-headers/prov-headers.html).
+- [HTTP 頭](/zh-TW/docs/Web/HTTP/Headers)
+  - : HTTP 消息頭用於描述資源或服務器或客戶端的行為。可以使用 `X-` 前綴添加自定義專有頭。其他的可以在 [IANA registry](https://www.iana.org/assignments/message-headers/message-headers.xhtml#perm-headers) 中找到，其原始定義在 [RFC 4229](https://tools.ietf.org/html/rfc4229)。 IANA 同時也維護著一份 [registry of proposed new HTTP message headers](https://www.iana.org/assignments/message-headers/message-headers.xhtml#prov-headers)。
 - [HTTP 請求方法](/zh-TW/docs/Web/HTTP/Methods)
-  - : 透過 HTTP 有幾種不同操作方法：{{HTTPMethod("GET")}}, {{HTTPMethod("POST")}}, and also less common requests like {{HTTPMethod("OPTIONS")}}, {{HTTPMethod("DELETE")}}, or {{HTTPMethod("TRACE")}}.
-- [HTTP 狀態回應碼](/zh-TW/docs/Web/HTTP/Response_codes)
-  - : HTTP response codes indicate whether a specific HTTP request has been successfully completed. Responses are grouped in five classes: informational responses, successful responses, redirections, client errors, and servers errors.
-- [CSP directives](/zh-TW/docs/Web/HTTP/Headers/Content-Security-Policy)
-  - : The {{HTTPHeader("Content-Security-Policy")}} response header fields allows web site administrators to control resources the user agent is allowed to load for a given page. With a few exceptions, policies mostly involve specifying server origins and script endpoints.
+  - : 透過 HTTP 有幾種不同操作方法：{{HTTPMethod("GET")}}, {{HTTPMethod("POST")}}，或是一些不太常見的請求方式，像是：{{HTTPMethod("OPTIONS")}}, {{HTTPMethod("DELETE")}}, or {{HTTPMethod("TRACE")}}.
+- [HTTP 狀態回應碼](/zh-TW/docs/Web/HTTP/Status)
+  - : HTTP 狀態碼用來表示特定的 HTTP 請求是否已成功完成。響應分為五類：消息響應，成功響應，重定向，客戶端錯誤和服務器錯誤。
+- [CSP 指令](/zh-TW/docs/Web/HTTP/Headers/Content-Security-Policy)
+  - : {{HTTPHeader("Content-Security-Policy")}} 響應報頭字段允許網站管理員控制頁面上哪些資源能夠被用戶代理程序加載。除了少數例外，此策略主要涉及指定服務器來源和腳本終端。
```

Comment:
- : The {{HTTPHeader("Content-Security-Policy")}} response header fields allows web site administrators to control resources the user agent is allowed to load for a given page. With a few exceptions, policies mostly involve specifying server origins and script endpoints.
```suggestion
  - : {{HTTPHeader("Content-Security-Policy")}} 回應檔頭讓網站管理員控制哪些頁面上的資源能被用戶端程式（user agent）載入。除了少數特例外，此政策主要關於指定來源伺服器和腳本程式的端點（endpoints）。
```